aesthetic_ants/penguin/player.py

```diff
@@ -0,0 +1,30 @@
+import pyglet
+import pyglet.window.key as key
+from utils import angle_between, loader, keys
```

Comment:
```suggestion
from .utils import angle_between, loader, keys
```

Move this down below too and move `from math import degrees` to the top`, so the imports should look like this:
```py
from math import degrees

import pyglet
import pyglet.window.key as key

from .utils import ...
```